src/Form/Form.js

```diff
@@ -428,14 +436,17 @@ class Form {
             }
 
             if (this.shouldOpenTooltip(e, input)) {
+                // On mobile and extensions we don't trigger the focus event to avoid
+                // keyboard flashing and conflicts with browsers' own tooltips
                 if (
-                    this.device.globalConfig.isMobileApp &&
+                    (this.device.globalConfig.isMobileApp || this.device.globalConfig.isExtension) &&
```

Comment:
I would use `this.device instanceOf ExtensionInterface` here, or add a `platform` property to each interface.  - mostly I say this because the logic for selecting the `ExtensionInterface` has already ran at this point. Please see the comment in `config.js`